src/styles/annotator/notebook.scss

```diff
@@ -0,0 +1,28 @@
+@use '../variables' as var;
+@use '../mixins/molecules';
+
+.notebook-outer {
+  // TODO: CSS namespace conflicts?
```

Comment:
We don't need to worry about that right away, but I expect it will come up at some point. Shadow DOM v1 support is pretty ubiquitous at this point so we might be able to rely mostly on that for everything except the outermost container element of the notebook, rather than use complex CSS specificity hacks.